assets/src/dashboard/components/table/index.js

```diff
@@ -37,6 +40,24 @@ export const TableHeader = styled.thead`
   border-bottom: ${({ theme }) => theme.table.border};
 `;
 
+export const StickyTableHeader = styled(TableHeader)`
+  width: 100%;
+  border-top: 0;
+  border-bottom: 0;
+  th {
+    height: 100%;
+    background: ${({ theme }) => theme.colors.gray50};
+    border-top-width: 0;
+    border-bottom: ${({ theme }) => theme.table.border};
```

Comment:
Tables are super fun and love their borders. I'm stripping them out here for the "sticky" version of the table header because it was making the top position off by 2 pixels.